static/js/modules/global.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
```

Comment:
How did this get here? Surely `global.js` isn't a module. From inspecting `package.json`, it looks like `modules/` is expected to only contain the global nav, and should be ignored from the codebase.